PimaxCrystalAdvanced.cs

```diff
@@ -64,13 +70,23 @@ public override void Update()
 
         var data = task.Result;
 
-        UnifiedTracking.Data.Eye.Left.Gaze =
-            data.Left.GazeDirectionIsValid ? ToVrcftVector2(data.Left.GazeDirection) : Vector2.zero;
-        UnifiedTracking.Data.Eye.Right.Gaze =
-            data.Right.GazeDirectionIsValid ? ToVrcftVector2(data.Right.GazeDirection) : Vector2.zero;
+        if (data.Left.GazeDirectionIsValid)
+            UnifiedTracking.Data.Eye.Left.Gaze = data.Left.GazeDirection.ToVRCFT().FlipXCoordinates();
```

Comment:
I don't agree, this causes eye jitter which makes the overall tracking experience worse for the user.

Reply:
To clarify, there is issue if the position is reset to zero, it will not properly handle "cross-eyes" because the Eye goes beyond "valid" range and then it resets, if we don't reset the position here then it will instantly regain valid data once Tobii can track it, this change only yields better results for the user.